extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfig.java

```diff
@@ -23,6 +26,14 @@ public interface HibernateOrmRuntimeConfig {
     @ConfigDocMapKey("persistence-unit-name")
     Map<String, HibernateOrmRuntimeConfigPersistenceUnit> persistenceUnits();
 
+    /**
+     * Configuration for enabling request scoped sessions
+     */
+    @ConfigDocSection
```

Comment:
This annotation does not make sense here, it's mainly for "composite" properties (with sub-properties).

```suggestion
```